docs/docker.md

```diff
@@ -0,0 +1,160 @@
+# Prepare docker image
+
+Please follow the docker part of
+[README.md](https://github.com/google/android-cuttlefish/blob/main/README.md#docker)
+in `google/android-cuttlefish` github repository, and check if
+`cuttlefish-linux-infrastructure` exists.
```

Comment:
should we explain how to check this?

Reply:
That's why the precondition of this PR is https://github.com/google/android-cuttlefish/pull/549. Sorry for not pushing second commit at there. README.md at there will explain how to check.